### PR TITLE
Added stubs

### PR DESCRIPTION
**Pull request by metavasko**

### CODE REVIEW

File: metavasko-London.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#LONDON
```
Comment (FrancescoBal) [inline metavasko-London.md:1]: London is EXPENSIVE!!!!

File: metavasko-London.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+### FOOD RECOMENDATIONS
```
Comment (angelotrivelli) [inline metavasko-London.md:3]: All good ideas, but can you name a specific place?

**Comment by FrancescoBal** [comment]
:eu:

**Review by FrancescoBal — APPROVED** [review]
All looks good!!!

**Pull request merged**